scripts/kubeadm_test.sh

```diff
@@ -50,28 +50,28 @@ set -e
 
 # Wait for CONTIV to start for up to 10 minutes
 sleep 10
-for i in {0..20}
-do
-  response=$(curl -k -H -s "Content-Type: application/json" -X POST -d '{"username": "admin", "password": "admin"}' https://$contiv_master:10000/api/v1/auth_proxy/login || true)
-  if [[ $response == *"token"* ]]; then
-    echo "Install SUCCESS"
-    echo ""
-    cat <<EOF
+for i in {0..20}; do
+	response=$(curl -k -H -s "Content-Type: application/json" -X POST -d '{"username": "admin", "password": "admin"}' https://$contiv_master:10000/api/v1/auth_proxy/login || true)
+
+	cat <<EOF
```

Comment:
Hmm, it looks like the logic was changed here... this heredoc was originally inside the outer `if` statement